app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+    @comments = @user.comments
+  end
```

Comment:
[OPTIONAL] [HIGHLY RECOMMENDED]

- [ ] Sorry, I have tagged wrong, Use `last_five_comments` method to render the post's 5 most recent comments in the post's index method.